### ISSUE TITLE
Phonegap: Login problem

### ISSUE THREAD

**Issue by kegita** [ccoenraets/OpenFB#62]
Hi! I have a problem with login and i am not able to resolve it. I have followed every step and i have inserted "https://www.facebook.com/connect/login_success.html"; after insert user data on Facebook Login, appears an error and, after click on "OK" button, the phone shows a white page with "success" label and other text about URL security.

**Comment by fanals** [comment]
You need to install the inappbrowser plugin:

https://github.com/ccoenraets/OpenFB#user-content-cordova-instructions

**Comment by kegita** [comment]
I have installed this plugin, but the problem there is again.

**Comment by TrollWarlord** [comment]
same

**Comment by AgentChris** [comment]
i have the same problem, the Android/web emulator works fine though

**Comment by TrollWarlord** [comment]
I've narrowed it down to ios only. 

Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by eugenemaning** [comment]
Please take time to look at the present issues before opening a new one. https://github.com/ccoenraets/OpenFB/issues/30#issuecomment-98092086

**Comment by sajclarke** [comment]
Hi, can you check your facebook app to ensure you have added the "website" platform?

In Facebook Developers Console, go to your app then go to "Basic" section and under the section with App ID, click "Add Platform" and choose "website". You do not have to add any of the URLs since you're using cordova.

Also make sure that you have turned on OAuth Login in the "Advanced" section

This worked for me. Let me know if it works for you

**Comment by ignacio-chiazzo** [comment]
> You do not have to add any of the URLs since you're using Cordova.????

 You cannot have a website platform without having a URL @sajclarke

**Comment by ndelvalle** [comment]
Yeah @ignacio-chiazzo is right, you can't leave it blank. Should it go `http://localhost/`?
